File: resources/android/src/com/victorycodedev/plugins/nativephp_fetch/events/FetchEvents.kt
package com.victorycodedev.plugins.nativephp_fetch

internal object FetchEvents {
    const val STARTED = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestStarted"
    const val COMPLETED = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestCompleted"
    const val FAILED = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestFailed"
    const val CANCELLED = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestCancelled"
    const val UPLOAD_PROGRESS = "Victorycodedev\\NativephpFetch\\Events\\FetchUploadProgress"
    const val DOWNLOAD_PROGRESS = "Victorycodedev\\NativephpFetch\\Events\\FetchDownloadProgress"
    const val DOWNLOAD_COMPLETED = "Victorycodedev\\NativephpFetch\\Events\\FetchDownloadCompleted"
    const val RETRYING = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestRetrying"
}

File: resources/ios/Sources/Events/FetchEvents.swift
import Foundation

enum FetchEvents {
    static let started = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestStarted"
    static let completed = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestCompleted"
    static let failed = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestFailed"
    static let cancelled = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestCancelled"
    static let uploadProgress = "Victorycodedev\\NativephpFetch\\Events\\FetchUploadProgress"
    static let downloadProgress = "Victorycodedev\\NativephpFetch\\Events\\FetchDownloadProgress"
    static let downloadCompleted = "Victorycodedev\\NativephpFetch\\Events\\FetchDownloadCompleted"
    static let retrying = "Victorycodedev\\NativephpFetch\\Events\\FetchRequestRetrying"
}
